apps/condo/domains/billing/schema/resolvers/accountResolver.js

```diff
@@ -28,7 +28,12 @@ class AccountResolver extends Resolver {
     }
 
     async init () {
-        this.accounts = await find('BillingAccount', { context: { id: this.billingContext.id }, deletedAt: null })
+        //this.accounts = await find('BillingAccount', { context: { id: this.billingContext.id }, deletedAt: null })
```

Comment:
Just delete the line if you don't need it
```suggestion
```

Reply:
will remove it later, needs this to make some tests